addon/components/nypr-accounts/basic-card.js

```diff
@@ -74,113 +72,147 @@ export default Component.extend({
     changeset.restore(snapshot);
   },
 
-  commit(changeset) {
+  checkForExistingAttribute: task(function * ({key, value, errorMessage}) {
+    let validator = validateRemote({
```

Comment:
this thing can probably be refactored out of a module and just live in this task